### PR TITLE
Move F# to consume Roslyn 2.6.0-vs-for-mac-62317-08

### PR DESCRIPTION
**Pull request by jasonmalinowski**
This carries along some breaking changes (dotnet/roslyn@efd59a3bf863d2516ae96c95678f8200bb816982) in Roslyn's Quick Info implementation that we must react to. Since it's our first move to the 2.6 line it also carries along some deprecation of some APIs (namely, classification and comment handling) that we also respond to as well.

### CODE REVIEW

**Comment by brettfo** [comment]
From the Jenkins log:
```
"D:\j\w\release_ci_pa---5a4853ab\vsintegration\tests\unittests\VisualFSharp.Unittests.fsproj" (Build target) (45) ->
(CoreCompile target) -> 
  D:\j\w\release_ci_pa---5a4853ab\vsintegration\tests\unittests\TestLib.LanguageService.fs(368,17): error FS0074: The type referenced through 'Microsoft.CodeAnalysis.Editor.QuickInfo.DeferredContentFrameworkElementFactory' is defined in an assembly that is not referenced. You must add a reference to assembly 'Microsoft.CodeAnalysis.EditorFeatures.Wpf'. [D:\j\w\release_ci_pa---5a4853ab\vsintegration\tests\unittests\VisualFSharp.Unittests.fsproj]
```

**Comment by jasonmalinowski** [comment]
@brettfo I find your lack of PackageReferences disturbing.

**Comment by brettfo** [comment]
Adding @KevinRansom as an FYI.  I've also turned off all auto-merges into and out of `dev15.6` since we'll need to be more deliberate now.

File: vsintegration/packages.config [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
-  <package id="Microsoft.CodeAnalysis.Common" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.EditorFeatures" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.EditorFeatures.Text" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.Features" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.VisualStudio.LanguageServices" version="2.3.0-beta2-61719-01" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.Common" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures.Text" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures.Wpf" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
```
Comment (T-Gro) [inline vsintegration/packages.config:14]: @jasonmalinowski :

Hi Jason, is the EditorFeatures.Wpf package still relevant in any way?
I am in the process of darc-ifiying our Roslyn dependencies (to spot issues earlier than when things are bound in VS) and I noticed this `Microsoft.CodeAnalysis.EditorFeatures.Wpf` has not been updated in years.

Which lead to me to this PR which added it despite not being used in the code....

Comment (jasonmalinowski) [reply vsintegration/packages.config:14]: It no longer exists. EditorFeatures vs. EditorFeatures.Wpf existed as a split when we had to also support VS for Mac and it's Cocoa-based editor, but that's no longer supported so we've merged the two packages back together again.

Comment (T-Gro) [reply vsintegration/packages.config:14]: That is lovely to hear, this package was source of manning NUGET conflict warnings I had to resolve.

Happy to remove and will see if I can in cascade also remove a lot of transient package references we did need to have in order to guide nuget around version mismatches 👍

**Pull request merged**